src/main/java/Information.java

```diff
@@ -0,0 +1,4 @@
+public class Information {
+    Task List[] = new Task[100];
+    int listLength = 0;
```

Comment:
You may want to rename listLength to something more intuitive as well, maybe taskCount or numberOfTasks, since list may be a bit generic.